code/game/gamemodes/miniantags/hades/hades.dm

```diff
@@ -65,163 +86,339 @@
-										sinPerson.name = H.name
-										sinPerson.real_name = H.real_name
-										var/datum/dna/lottery = H.dna
-										lottery.transfer_identity(sinPerson, transfer_SE=1)
-										sinPerson.updateappearance(mutcolor_update=1)
-										sinPerson.domutcheck()
-						else
-							src.say("Your sin will be punished, [sinPerson]!")
-							var/sinPersonspecies = pick(species_list)
-							var/newtype = species_list[sinPersonspecies]
-							sinPerson.set_species(newtype)
-					if("Sloth")
-						src.say("Your sin, [sinPerson], is Sloth.")
-						if(prob(50))
-							src.say("I will indulge your sin, [sinPerson].")
-							sinPerson.drowsyness += 50
-						else
-							src.say("Your sin will be punished, [sinPerson]!")
-							sinPerson.reagents.add_reagent("frostoil", 50)
-					if("Wrath")
-						src.say("Your sin, [sinPerson], is Wrath.")
-						if(prob(50))
-							src.say("I will indulge your sin, [sinPerson].")
-							sinPerson.reagents.add_reagent("bath_salts",100)
-						else
-							src.say("Your sin will be punished, [sinPerson]!")
-							sinPerson.reagents.add_reagent("lexorin", 100)
-							sinPerson.reagents.add_reagent("mindbreaker", 100)
+			if(prob(20))
+				var/chosenDefend = rand(1,3)
+				switch(chosenDefend)
+					if(1)
+						attacker.visible_message("<span class='warning'>[attacker] is lifted from the ground, shadowy powers tossing them aside.</span>")
+						attacker.throw_at_fast(pick(orange(src,7)),10,1)
+					if(2)
+						attacker.visible_message("<span class='warning'>[attacker] crackles with electricity, a bolt leaping from [src] to them.</span>")
+						sinLightning.Bolt(src,attacker,30,5,src)
+					if(3)
+						src.visible_message("<span class='warning'>[src] points his staff at [attacker], a swarm of eyeballs lurching fourth!</span>")
+						for(var/i in 1 to 4)
+							var/mob/living/simple_animal/hostile/carp/eyeball/E = new/mob/living/simple_animal/hostile/carp/eyeball(pick(orange(attacker,1)))
+							E.faction = faction
+							spawn(150)
+								if(E)
+									E.gib()
+
+/mob/living/simple_animal/hostile/hades/proc/sinShed(var/turf/T)
+	var/obj/effect/overlay/temp/cult/sparks/S = new/obj/effect/overlay/temp/cult/sparks(T)
+	S.density = 0
+	S.anchored = FALSE
+	S.throw_at_fast(src,10,1)
+	var/obj/effect/overlay/temp/bloodsplatter/BS = new/obj/effect/overlay/temp/bloodsplatter(T)
+	BS.density = 0
+	BS.anchored = FALSE
+	BS.throw_at_fast(src,10,1)
+
+/mob/living/simple_animal/hostile/hades/proc/Transfer(var/mob/living/taken, var/turf/transferTarget)
+	if(transferTarget)
+		playsound(get_turf(taken), 'sound/magic/Ethereal_Enter.ogg', 50, 1, -1)
+		var/atom/movable/overlay/animation = new /atom/movable/overlay(get_turf(taken))
+		animation.name = "water"
+		animation.density = 0
+		animation.anchored = 1
+		animation.icon = 'icons/mob/mob.dmi'
+		animation.layer = FLY_LAYER
+		animation.master = src
+		animation.icon_state = "liquify"
+		flick("liquify",animation)
+		spawn(15)
+			qdel(animation)

```

Comment:
Make a new `/obj/effect/overlay/temp` subtype for this with appropriate variables, please.